models/reporting/olids/person_status/dim_person_residence.sql

```
@@ -0,0 +1,42 @@
+        count (*) as household_registered_count,
+        sum (frail_flag) as frail_household_count
+    from frailty_table
+    group by uprn_hash
+)
+
+select bs.person_id
+    , bs.sk_patient_id
+    , bs.uprn_hash
+    , hs.household_registered_count
+    , hs.frail_household_count
+    , case when hs.frail_household_count =  hs.household_registered_count then 1 else 0 end as frail_household_flag
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)